app/controlplane/internal/biz/workflowrun.go

```diff
@@ -256,22 +256,19 @@ func (uc *WorkflowRunUseCase) SaveAttestation(ctx context.Context, id string, en
 	return uc.wfRunRepo.SaveAttestation(ctx, runID, envelope, digest)
 }
 
+type RunListFilters struct {
```

Comment:
this filter object will help us extend this call in the future i.e by filtering by runner type.